lib/quantaq-modulair-x/quantaq-modulair-x-converter.ts

```diff
@@ -0,0 +1,110 @@
+import { JtsDocument, TimeSeries } from '@eagle-io/timeseries'
+import { Converter } from '../../converter'
+
+/**
+ * Convert the output from QuantAQ's RESTful API for the MODULAIR-X and MODULAIR-X-PM
+ * product. Note: MODULAIR-X-UFP is not yet supported.
+ *
+ * Docs: https://docs.quant-aq.com/api
+ */
+export class QuantAQModulairXConverter extends Converter {
+  convert (input: Buffer): JtsDocument {
+    const snSeries = new TimeSeries({ name: 'sn', type: 'TEXT' })
+
+    // pm data
+    const pm1Series = new TimeSeries({ name: 'pm1', type: 'NUMBER' })
+    const pm25Series = new TimeSeries({ name: 'pm25', type: 'NUMBER' })
+    const pm10Series = new TimeSeries({ name: 'pm10', type: 'NUMBER' })
+    const tspSeries = new TimeSeries({ name: 'tsp', type: 'NUMBER' })
+
+    // gas data
+    const noSeries = new TimeSeries({ name: 'no', type: 'NUMBER' })
+    const no2Series = new TimeSeries({ name: 'no2', type: 'NUMBER' })
+    const o3Series = new TimeSeries({ name: 'o3', type: 'NUMBER' })
+    const coSeries = new TimeSeries({ name: 'co', type: 'NUMBER' })
+
+    // weather station data
+    const wsSeries = new TimeSeries({ name: 'wx_ws', type: 'NUMBER' })
+    const wdSeries = new TimeSeries({ name: 'wx_wd', type: 'NUMBER' })
+    const wsScalarSeries = new TimeSeries({ name: 'wx_ws_scalar', type: 'NUMBER' })
+    const uSeries = new TimeSeries({ name: 'wx_u', type: 'NUMBER' })
+    const vSeries = new TimeSeries({ name: 'wx_v', type: 'NUMBER' })
+    const pressureSeries = new TimeSeries({ name: 'wx_pressure', type: 'NUMBER' })
+    const tempSeries = new TimeSeries({ name: 'wx_temp', type: 'NUMBER' })
+    const rhSeries = new TimeSeries({ name: 'wx_rh', type: 'NUMBER' })
+    const dewPointSeries = new TimeSeries({ name: 'wx_dew_point', type: 'NUMBER' })
+
+    // Get the records
+    const records = JSON.parse(input.toString())
+
+    // Iterate over each record and convert to eagle-io's format
+    records.data.forEach(
+      (row: {
+        timestamp: string;
+        sn: string;
+        pm1: string;
+        pm25: string;
+        pm10: string;
+        tsp: string;
+        no: string;
+        no2: string;
+        o3: string;
+        co: string;
+        met: { wx_ws: string; wx_wd: string; wx_ws_scalar: string; wx_u: string; wx_v: string; wx_pressure: string; wx_temp: string; wx_rh: string; wx_dew_point: string; };
+      }) => {
+        const ts = this.dayjs.tz(row.timestamp, 'UTC')
```

Comment:
in the other file, we're doing this without the UTC stuff. when I tried to run a test on this doing it that way, it failed, which has troubling implications for the existing integrations...